### PR TITLE
ショッピングカートから商品を削除する機能を修正しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java [hunk 1]
```diff
@@ -16,21 +16,21 @@
...
-		if (orderItemId == null) {
-			return "redirect:/showShoppingCart";
-		}
-		deleteShoppingCartService.deleteShoppingCart(orderItemId);
-		return "redirect:/showShoppingCart";
-	}
+  /**
+   * ショッピングカートから商品を削除します.
+   * 
+   * @param orderItemId 削除する注文商品のID
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/DeleteShoppingCartController.java:25]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

File: src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public class SecurityConfig {
 	SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		// アクセスの制限に関する設定
 		http.authorizeHttpRequests(ahr -> ahr
-				.requestMatchers("/showItemList/**", "/registerUser/**", "/login/**", "/ShowItemDetail/**",
+				.requestMatchers("/showItemList/**", "/registerUser/**", "/login/**", "/showItemDetail/**",
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:27]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 123).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/SecurityConfig.java:27]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 3]
```diff
@@ -57,7 +58,8 @@ public List<Item> findByName(String name) {
   /**
    * 商品IDによってアイテム（おもちゃ）を取得するメソッド.
    * 
-   @param id 商品ID
+   * 
+   * @param id 商品ID
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

**Review by yamada-kensyu — APPROVED** [review]
承認します
ありがとうございます

**Pull request merged**